packages/backend/src/plugins/auth.ts

```diff
@@ -1,56 +0,0 @@
-import {
```

Comment:
@schultzp2020 are these files not needed or ?

Reply:
These files are no longer needed with the new backend.